notebooks/02.08-Sorting.ipynb

```diff
@@ -30,7 +30,7 @@
    "cell_type": "markdown",
```

Comment:
Intro is nice. Can we make the book logo centerd at the top? <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=0>ReviewNB</a>_

Reply:
Why would I want to do that? **Instead checkout this statement that I have written in bold.** 

I care about following:

- A bullet list
- in the comment
- need to render properly

Also, checkout this hyperlinked [article](https://medium.com/) on medium.

@amit1rrr Now it's your turn to make sure at mentions and PR/issue references are rendering fine.  #3

> This is a quote

Following is a code snippet
```suggestion
   "cell_type": "markdown",
```

Reply:
You have given a lot of different types of content to render. But don't worry, I got it under control. Here's some random text (source):

Now that the config is in `values.yaml` we are not using application flags for cert-manager.
I think it's a still a good idea to keep this functionality so apps in future can pass helm installation flags via this arg. Thoughts?

Reply:
Pretty good. Now I will add to your woes. Here's a task list.

- [ ] Task 1 to be done

- [ ] Task 2 to be done

Reply:
"This is commenty from Postman for a newly create API"

Reply:
<p>a</p>

Reply:
This is a comment via API

Reply:
"This is commenty from Postman for a newly create API"

Reply:
"This is commenty from Postman for a newly create API"

Reply:
"This is commenty from Postman for a newly create API"

Reply:
"This is commenty from Postman for a newly create API"

Reply:
<p>Let's post something.</p>

Reply:
<p>Let's post something.</p>

Reply:
<p>Post something 2.</p>

Reply:
<p>Post something 2.</p>

Reply:
<p>Post something 3</p>

Reply:
<p>Post something 4</p>

Reply:
<p>Post <a href="https://reviewnb.com" target="_blank">something</a> 4</p><p><br></p><ul><li>Bullet 1</li><li>2</li><li>3</li></ul><p><br></p><p>I am <strong>bold</strong></p>

Reply:
<p>abcd</p>